lib/search_solr_tools/config/environments.rb

```diff
@@ -5,10 +5,10 @@
 module SearchSolrTools
   # configuration to work with solr locally, or on integration/qa/staging/prod
   module SolrEnvironments
-    YAML_ENVS = YAML.load_file(File.expand_path('environments.yaml', __dir__))
+    YAML_ENVS = YAML.load_file(File.expand_path('environments.yaml', __dir__), aliases: true)
 
     def self.[](env = :development)
-      YAML_ENVS[:common].merge(YAML_ENVS[env.to_sym])
+      YAML_ENVS[env.to_sym]
```

Comment:
This change, coupled with the change in environments.yaml, seems to have the same net effect but in a cleaner way.